### PR TITLE
changed naming for distance to branch length

### PR DESCRIPTION
**Pull request by albert-smit**

### CODE REVIEW

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java [hunk 1]
```diff
@@ -26,6 +26,11 @@
      * the default color for this segment.
      */
     private static final Color DEFAULT_COLOR = Color.RED;
+    
+    /**
+     * The default name for distance.
+     */
+    private static final String Distance_Name = "Branch Length: ";
```
Comment (rutgervandenberg) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java]: uppercase

**Comment by rutgervandenberg** [comment]
Looks good.

**Comment by jorenham** [comment]
:+1:

**Comment by joswinter** [comment]
Nice!

**Comment by alangerak** [comment]
Looks good

**Pull request merged**